### ISSUE TITLE
Make ace medical basic/adv be decided from parameters

### ISSUE THREAD

**Issue by Cuel** [fparma/comfy-downtime#1]
I don't see the point of having 2 missions

**Comment by ComfyPillow** [comment]
Yes, it's an annoyance that I didn't have time to properly address.
I was worried initially that if I configure both ADV Settings and basic Medical settings, but keep Basic medical default that it would have some unintended side-effects.

From what I've seen, setting the following global var should allow for switching the Medical Level

``` c
ace_medical_level = 1; // Basic Medical
ace_medical_level = 2; // Advanced Medical
```

I don't know how early in the init these have to be set in order to get applied correctly and not cause issues. Maybe it doesen't matter tho.

Build Script can be easily Adapted to only build 1 mission for each map tho, and that would reduce a bit of complexity from the repo.
